.dependabot/config.yml

```diff
@@ -0,0 +1,5 @@
+version: 1
+update_configs:
+  - package_manager: "dotnet:nuget"
+    directory: "/"
+    update_schedule: "daily"
```

Comment:
```suggestion
    update_schedule: "daily"
    target_branch: "master"
```

Reply:
Unless we're happy to use the default branch for all repos?

Reply:
We should be using the default branch for all repos. That should be equal to `master` in all cases currently as well, so there's no reason to specify it.

If we do have a repo that doesn't have `master` as the default branch, then dependabot would be looking at that default branch for this config file, and then be told to update a different branch, which seem like behavior we would want.